### PR TITLE
CH-64 and CH-63 fix: Assign 'CH admin miscellaneous' and 'assign CH Fund' to administrator role

### PR DESCRIPTION
**Pull request by monishdeb**

### CODE REVIEW

File: chfunds.permitted-roles.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This constant DRUPAL_ROLE_PERMISSIONS holds the drupal role and the respective with CH Permissions in serialised format
```
Comment (JoeMurray) [inline chfunds.permitted-roles.php:3]: Copyedit change to 'This constant DRUPAL_ROLE_PERMISSIONS holds an array of drupal roles and permissions that are to be programmatically assigned to each role.'

File: chfunds.permitted-roles.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * unserialize(DRUPAL_ROLE_PERMISSIONS) = [
+ *   'administrator' => [
+ *     'CH admin miscellaneous',
+ *     'assign CH Fund',
+ *    ],
+ *  ];
+ *
+ * which means 'adminstrator' role is given 'CH admin miscellaneous' and 'assign CH Fund' permission.
+ * You can change it to whichever role(s) you like to assign permission with. Like if you want to give
```
Comment (JoeMurray) [inline chfunds.permitted-roles.php:15]: Copyedit suggestion: 'You can change it to assigning whichever permissions you want to whichever roles you want, assuming permissions and roles exist. Like if you want to give'

**Comment by JoeMurray** [comment]
Sorry, but could @monishdeb or @irfanhaq1 update the PR with my changes, or create a new commit to include them? Thx.

File: chfunds.permitted-roles.php (hunk 1 above)
Comment (monishdeb) [reply chfunds.permitted-roles.php:3]: Done. Submitted PR https://github.com/CanadaHelps/chfunds/pull/8

**Pull request merged**